monitoring/persistent_stats_history.cc

```diff
@@ -0,0 +1,171 @@
+// Copyright (c) 2011-present, Facebook, Inc.  All rights reserved.
+//  This source code is licensed under both the GPLv2 (found in the
+//  COPYING file in the root directory) and Apache 2.0 License
+//  (found in the LICENSE.Apache file in the root directory).
+// Copyright (c) 2011 The LevelDB Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file. See the AUTHORS file for names of contributors.
+
+#include "monitoring/persistent_stats_history.h"
+
+#include <cstring>
+#include <string>
+#include <utility>
+#include "db/db_impl/db_impl.h"
+#include "port/likely.h"
+#include "util/string_util.h"
+
+namespace rocksdb {
+// 10 digit seconds timestamp => [Sep 9, 2001 ~ Nov 20, 2286]
+const int kNowSecondsStringLength = 10;
+const int kMicrosInSecond = 1000 * 1000;
+const std::string kFormatVersionKeyString =
+    "__persistent_stats_format_version__";
+const std::string kReaderVersionKeyString =
+    "__persistent_stats_reader_version__";
```

Comment:
Can we rename it from "reader version" to something like "compatible version" or something like that?